podcasts/Onboarding/Plus/Account Prompt/UpgradePrompt.swift

```diff
@@ -0,0 +1,61 @@
+import SwiftUI
+import PocketCastsServer
+
+struct UpgradePrompt: View {
```

Comment:
Can we name it `UpgradePromptView`? There are some other views without the suffix `View` already but I wonder if that makes it more descriptive? 🤔

Reply:
I can do, this is something that even Apple is not very consistent. They have loads of controls that are not named view: Button, Text, etc...

Reply:
Oh, that makes sense. Nevermind!